data-processed/ECDC-hosp_model/2021-11-22-ECDC-hosp_model.csv

```diff
@@ -0,0 +1,1633 @@
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.65,4155
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.65,18860
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.65,26238
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.65,28206
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.7,4219
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.7,19151
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.7,26642
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.7,28640
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.75,4285
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.75,19450
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.75,27058
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.75,29088
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.8,4365
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.8,19813
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.8,27563
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.8,29630
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.85,4502
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.85,20436
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.85,28430
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.85,30562
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.9,4573
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.9,20759
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.9,28879
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.9,31045
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.95,4805
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.95,21809
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.95,30340
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.95,32615
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.975,5114
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.975,23210
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.975,32289
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.975,34711
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"GR","quantile",0.99,5299
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"GR","quantile",0.99,24051
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"GR","quantile",0.99,33459
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"GR","quantile",0.99,35969
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"IS","point",NA,22
+"2021-11-22","2 wk ahead inc hosp",2021-12-04,"IS","point",NA,23
+"2021-11-22","3 wk ahead inc hosp",2021-12-11,"IS","point",NA,23
+"2021-11-22","4 wk ahead inc hosp",2021-12-18,"IS","point",NA,22
+"2021-11-22","1 wk ahead inc hosp",2021-11-27,"IS","quantile",0.01,NA

```

Comment:
It looks like there are NA values for Iceland, and this is causing the validation check failure. If the model produces NA, please could you remove those forecast predictions (or replace with 0, whatever is appropriate)?

Reply:
(thanks due to Hugo @Bisaloo - I haven't seen this error with NA values before and the R validations check neatly caught out the lines where it errored ⚡👏 )